columns/stories/index.stories.js

```diff
@@ -0,0 +1,54 @@
+import '~/columns/dockit-columns.define';
+import '~/do-dont/define-all.js';
+
+export default {
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+export const columns = () => /*html*/ `
+  <dockit-columns>
+    <dockit-do>
+      <ul slot="instructions">
+        <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
+        <li>
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+        </li>
+        <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
+      </ul>
+    </dockit-do>
+    <dockit-dont>
+      <ul slot="instructions">
+        <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit</li>
+        <li>
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+        </li>
+        <li>Duis aute irure dolor in reprehenderit in voluptate velit</li>
+      </ul>
+    </dockit-dont>
+  </dockit-columns>
+`;
+
+export const columns_custom_gap = () => /*html*/ `
+  <dockit-columns style="gap: 0.5rem;">
```

Comment:
I would have `gap` as an attribute of  `dockit-columns` 
using style gap is a bit of a hack because you need to know this is a flexbox underneath...